projects/kserve/testing/models/watsonx_base.yaml

```diff
@@ -0,0 +1,11 @@
+serving_runtime:
+  kserve:
+    resource_request:
+      cpu: 8
+    extra_env:
+      TRANSFORMERS_CACHE: "/tmp/transformers_cache"
+inference_service:
+  model_format: "pytorch"
+  min_replicas: 1
+  maxReplicas: 1
+  single_container: true
```

Comment:
can you move this line to the beginning of `serving_runtime`?
```
serving_runtime:
  single_container: true
  kserve:
```
(it's a ServingRuntime properly, not ISVC, I don't know how I got that wrong in my first try 🤦🏻 )

Could you also rename this file to `base_....yaml`? I'm not sure what base name would accurately describe this file, maybe `tgis-only`? 

and `maxReplicas: 1` property does not exist 🙃 (please remove it from here)

Reply:
Fixed.